src/site/stages/build/process-cms-exports/schemas/common/media.js

```diff
@@ -6,5 +6,5 @@ module.exports = {
   // Adding the catch-all object schema is a temporary fix
   // until we have schemas & transformers for all
   // media entities
-  anyOf: getAllSchemasOfType('media').concat([{ type: 'object' }]),
+  anyOf: getAllSchemasOfType('media').concat([{ type: ['object', 'null'] }]),
```

Comment:
I'm actually not sure I like this. I think the alternative is to use a `oneOf` schema one level up:
```js
fieldMedia: {
  oneOf: [
    { $ref: 'Media' },
    { type: 'null' }
  ]
}
```